lib/commands/extract.js

```diff
@@ -502,6 +502,16 @@ EXTRACTING TEMPLATE TRANSLATIONS
         shell.exec(`${parser} ${nodeArgs}`);
       });
 
+      // Fix paths for Windows - there, xgettext-template makes paths like ./app\components\...
+      let fileContent = fs.readFileSync(tmpPOTFile, encoding);
+
+      let regex = /#: (.*)(\\)(.*)\.hbs/gm;
+      while (regex.exec(fileContent)) {
+        fileContent = fileContent.replace(regex, '#: $1/$3.hbs');
+      }
+
+     fs.writeFileSync(tmpPOTFile, fileContent);
+
```

Comment:
I'm not happy with that change, it introduces another file I/O and the HBS extraction is already very slow. The comments are for information only and window paths are backslashs. So I would suggest to normalize the paths in the extraction methods, so you wouldn't get a mixture. Hope that's fine - please also check this again on your machine.